spacepy/pybats/bats.py

```diff
@@ -863,18 +877,82 @@ def calc_j(self):
 
         self['j'] = sqrt(self['jx']**2.0 + self['jy']**2.0 + self['jz']**2.0)
         self['j'].attrs = {'units':self['jx'].attrs['units']}
-    
+
+    @calc_wrapper
+    def calc_uperp(self):
```

Comment:
These new functions should have a versionadded directive and ultimately be mentioned in the release notes. Can you also provide a little more information in the class docs on how these are used? What would be _great_ would be to actually get the autodoc working and have the methods properly linked in a table, but that has not been the case in pybats so far, so I think it makes more sense to do a documentation overhaul later.

Reply:
Versionadded: Done.
Expanded class docs: Done.
autodoc and release note stuff: TBD (I'll update release notes after #652 rebase)